client/my-sites/stats/all-time-highlights-section/style.scss

```diff
@@ -1,80 +1,81 @@
-$header-font: Recoleta, sans-serif;
+@import "@automattic/typography/styles/variables";
+$header-font: Recoleta, $sans;
 
 .stats__all-time-highlights-section {
```

Comment:
These styles were bleeding onto the 7-day highlights (and probably onto annual highlights), so I limited the scope of these styles to the all time highlights section. Let's try our best to avoid regressions like this in the future (cc @dognose24).